zcash_client_backend/src/data_api.rs

```diff
@@ -822,6 +885,17 @@ pub trait InputSource {
         exclude: &[Self::NoteRef],
     ) -> Result<SpendableNotes<Self::NoteRef>, Self::Error>;
 
+    /// Returns metadata describing the structure of the wallet for the specified account.
+    ///
+    /// The returned metadata value must exclude spent notes and unspent notes having value less
+    /// than the specified minimum value or identified in the given exclude list.
```

Comment:
This is a good change, because I initially misinterpreted it as "(spent notes and unspent notes) having ..." and wondered why that was not just "notes having ...".